src/main/java/salesforce/ui/pages/AppPageFactory.java

```diff
@@ -66,27 +70,31 @@ public static AbstractCampaignListPage getCampaignsPage() {
     }
 
     /**
-     * Returns a Opportunities page.
+     * Allows to instance Opportunities page.
      *
      * @return OpportunitiesPage instance.
      */
     public static AbstractOpportunitiesPage getOpportunitiesPage() {
-        if (userExperience.equals("Classic")) {
+        if (UserExperience.USER_EXPERIENCE_CLASSIC.get().equals(userExperience)) {
             return new OpportunitiesClassicPage();
+        } else if (UserExperience.USER_EXPERIENCE_LIGHTNING.get().equals(userExperience)) {
+            return new OpportunitiesLightningPage();
         }
-        return new OpportunitiesLightningPage();
+        return null;
```

Comment:
All of the would be e.g.: 
       if (userExperience.equals("Classic")) {
            return new NewCampaignClassicPage();
        }
        return new NewCampaignLightningPopUp();

Reply:
that case is not completly valid

Reply:
You are true, it would be a CONSTANT